apps/web/src/routes/_layout.svelte

```diff
@@ -41,7 +85,7 @@
 {#if isBrowser() !== true}
   <div class="pageloader is-active"><span class="title">LibreLingo</span></div>
 {/if}
-
-<main>
+<!-- Plase advice where you need to apply the styles to body or here in main  -->
```

Comment:
I would say this is the best place if it works !

Reply:
ok got it 👍